lib/holidays.rb

```diff
@@ -122,6 +122,40 @@ def between(start_date, end_date, *options)
       UseCaseFactory.between.call(start_date, end_date, date_driver_hash, regions, observed, informal)
     end
 
+    # Get next holidays occuring form date, inclusively.
+    #
+    # Returns an array of hashes or nil.
+    #
+    # Each holiday is returned as a hash with the following fields:
```

Comment:
This line is confusing based on what is below. Aren't the next few lines spelling out the incoming arguments, not what we are returning?

Other than this the documentation is great!